controllers/dspipeline_params.go

```diff
@@ -105,6 +105,10 @@ func passwordGen(n int) string {
 	return string(b)
 }
 
+func BoolPointer(b bool) *bool {
+	return &b
```

Comment:
can you move this to [util.go](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/controllers/util/util.go)

Reply:
I was considering using a one-liner anonymous function for this but this makes it easily readable so moved this to util as suggested.